.github/workflows/all-in-one-build.yml

```
@@ -0,0 +1,50 @@
+name: Docker
```

```suggestion
name: Build all-in-one
```
due to https://github.com/jaegertracing/jaeger/issues/2645#issuecomment-735256696

---

Later on I think we should merge this into the overall `Docker Build + Deploy` workflow. Building the UI is one of the longest parts there, and UI needs to be built for query service as well, so there's really no reason for all-in-one to be separate like that. But we can address it in the PR that does the rest of the builds.